docs/advanced/atomic-swap.md

```diff
@@ -0,0 +1,38 @@
+# Atomic Swaps
+
+---
+
+## Overview
+
+An Atomic Swap is a smart contract technology which makes possible to exchange coins from two different blockchains without having to trust any third party, for example a centralized exchange. 
+For an atomic swap, a cryptocurrency must have the 'OP_CHECKLOCKTIMEVERIFY' opcode, which makes a timelock possible.
+The other prerequisite is that they need to have the same hash function. That doesn't necessarily mean that they use the same hash functions for mining, but they need to have the same hash function available in their transaction scripting language.
```

Comment:
Rather than making a vague statement and then immediately clarifying with detail, can we just begin with a detailed statement?

E.G.
> The other prerequisite is that they need to have a shared hash function available for use in their transaction scripting languages. For example, Decred supports blake256, sha256, ripemd160, and sha1.